hbase-server/src/test/java/org/apache/hadoop/hbase/quotas/TestQuotaUserOverride.java

```diff
@@ -78,39 +78,32 @@ public static void tearDownAfterClass() throws Exception {
   @Test
   public void testUserGlobalThrottleWithCustomOverride() throws Exception {
     final Admin admin = TEST_UTIL.getAdmin();
-    final String userOverrideWithQuota = User.getCurrent().getShortName() + "123";
+    final String userOverrideWithQuota = User.getCurrent().getShortName();
```

Comment:
I made a few small test changes to ensure that refreshes were called appropriately. Because of the logic in `ThrottleQuotaTestUtil.triggerUserCacheRefresh` assuming that we're waiting on throttling for the current user, I also had to change around the procedure of this test a bit to ensure that the current user is who we're throttling (rather than the current user plus a suffix, like we were before). But the logic is still sound